contributors/guide/github-workflow.md

```diff
@@ -74,6 +74,8 @@ git checkout -b myfeature
 Then edit code on the `myfeature` branch.
 
 #### Build
+This workflow is process-specific; for quick start build instructions for [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) please [see here](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#building-kubernetes-on-a-local-osshell-environment).
```

Comment:
The first link should use `https://git.k8s.io/kubernetes`
For the second link, you can just use a relative link `/contributors/devel/development.md#building-kubernetes-on-a-local-osshell-environment` (double check me on this I'm really bad at links 😛 )